rocky/rocky/views/organization_list.py

```diff
@@ -18,3 +28,32 @@ def get_queryset(self) -> list[Organization]:
             .prefetch_related("tags")
             .filter(id__in=[organization.id for organization in user.organizations])
         )
+
+    def post(self, request: HttpRequest, *args, **kwargs) -> HttpResponse:
+        """Perform actions based on action type"""
+        if request.POST.get("action") == "recalculate":
+            if not self.request.user.has_perm("tools.can_recalculate_bits"):
+                raise PermissionDenied()
+            organizations = self.request.user.organizations
+            number_of_bits = 0
+            failed = []
+            start_time = datetime.now()
+            for organization in organizations:
+                try:
+                    number_of_bits += OctopoesAPIConnector(settings.OCTOPOES_API, organization.code).recalculate_bits()
+                except Exception as exc:
+                    failed += [f"{organization}, ({str(exc)})"]
+                    logging.warning("Failed recalculating bits for %s, %s", organization, exc)
+            duration = datetime.now() - start_time
+            message = f"Recalculated {number_of_bits} bits for {len(organizations)-len(failed)} organizations."
+            message += f" Duration: {duration}."
+            if failed:
+                message += f"\nFailed for {len(failed)} organisations: {', '.join(failed)}"
+            messages.add_message(
+                request,
+                messages.INFO,
+                _(message),
```

Comment:
The problem here is that this message won't be translated. This is mainly because the message is pre-formatted, and you would need to use Django's gettext format. You might consider using the template to render this using different parts (as context data) for the translations (and use a short simple warning message, e.g., 'Recalculated bits for %d organizations').

Reply:
Maybe this should be raised as a separate issue, as the problem also exists for the 'normal' bit recalculation right?

Reply:
https://github.com/minvws/nl-kat-coordination/issues/3459

Reply:
Yes you're right, didn't notice it at first. But the one in `organization_settings.py` is much easier to tackle:
Instead of `messages.add_message(request, messages.INFO, _(f"Recalculated {number_of_bits} bits. Duration: {duration}"))` it should be something like: `messages.add_message(request, messages.INFO, _("Recalculated %d bits. Duration: %s", number_of_bits, duration))`.

I'm fine with fixing this in #3459 
`